giraffe/src/components/SimpleTable/PaginationNav/PaginationDirectionItem.tsx

```diff
@@ -22,18 +31,43 @@ export const PaginationDirectionItem = forwardRef<
   PaginationDirectionItemProps
 >(
   (
-    {testID = 'pagination-direction-item', direction, onClick, isActive},
+    {
+      id,
+      style,
+      testID = 'pagination-direction-item',
+      className,
+      direction,
+      onClick,
+      size = ComponentSize.Medium,
+      isActive,
+    },
     ref
   ) => {
+    const paginationClassName = classnames('cf-pagination--item--container', {
+      [`${className}`]: className,
+    })
+      .split(' ')
+      .reduce((accum, current) => styleReducer(styles, accum, current), '')
```

Comment:
doesn't `classnames` do the same thing the `styleReducer` does?

Reply:
No. The `styleReducer` changes the string to a unique string, while `classnames` still has the readable, original strings concatenated together. What the `styleReducer` is doing is actually reaching into the `styles` object that is imported and getting the unqiue string back.